src/main/java/com/blamejared/crafttweaker/impl_native/loot/ExpandLootContext.java

```diff
@@ -168,14 +170,13 @@ public static int getLootingModifier(final LootContext internal) {
     }
 
     /**
-     * Gets the world where the interaction happened, if it exists or can be obtained; null otherwise.
+     * Gets the world where the interaction happened.
```

Comment:
So this looks fine to me, but if I remember correctly, @TheSilkMiner has said that the world sometimes doesn't exist, so just going to ping him so he knows the doc is changing.

I took a look at the code, a world will always exist as it is required to get the loot table manager.